py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -1165,6 +1165,57 @@ def aten_ops_log(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.log2.default)
+def log2(
```

Comment:
I just noticed the naming was inconsistent. Can you modify the function name to `aten_ops_log2`?